WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Cells/ShippingLabelSummaryTableViewCell.swift

```diff
@@ -68,6 +74,37 @@ final class ShippingLabelSummaryTableViewCell: UITableViewCell {
         onButtonTouchUp?()
     }
 
+    func setPackageRates(_ rates: [String]) {
+        if rates.isEmpty {
+            packageRatesStackViews.forEach { stackView in
+                mainStackView.removeArrangedSubview(stackView)
+            }
+        } else {
```

Comment:
I think we need to remove all of the `packageRatesStackViews` first, every time we set the package rates in the summary cell (whether or not `rates` is empty). Then, we can set `packageRatesStackViews` if `rates` is not empty.

Otherwise, if you select rates for multiple packages, and then later change the selected rates, your first selection still shows up in the summary cell:

<img src="https://user-images.githubusercontent.com/8658164/135841388-f0c167be-c094-4f0e-945e-c3b67f62c6dd.png" width="300px">